dataprofiler/profilers/order_column_profile.py

```diff
@@ -209,9 +218,9 @@ def _merge_order(
         ) or order == "random":
             piecewise = False
 
-        return order, first_value, last_value, piecewise
+        return order, cast(int, first_value), cast(int, last_value), piecewise
```

Comment:
Does this need to be casted or is optional an acceptable output arg?

Reply:
I think `first_value` and `last_value` are always not None because one of the conditionals on lines 205 to 213 will run.